pom.xml

```diff
@@ -168,6 +173,18 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.3.0</version>
+        <configuration>
+          <mainClass>eu.dissco.core.translator.maven.MavenRunner</mainClass>
```

Comment:
When I try to run this plugin, I get 

> java.lang.ClassNotFoundException: eu.dissco.core.translator.maven.MavenRunner

Does this work for you?

Reply:
Yeah, it first needs a compile `mvn compile`. Long term there might be nicer ways for now... it does the job.